### PR TITLE
[FIX] Added markdown to custom status in UserInfo

### PR DESCRIPTION
**Pull request by yash-rajpal**
- [x] I have read the Contributing Guide 
  - [x] I have signed the CLA 
  - [x] Lint and unit tests pass locally with my changes
  - [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
  - [ ] I have added necessary documentation (if applicable)
  - [ ] Any dependent changes have been merged and published in downstream modules

## Proposed changes (including videos or screenshots)

Added markdown to custom status in Memberlist/userInfo. 

## Issue(s)

N/A
## Further comments

image

### CODE REVIEW

**Comment by yash-rajpal** [comment]
@ggazzo @MartinSchoeler Please Review. :)

**Comment by yash-rajpal** [comment]
@ggazzo @dougfabris Please have a look, Thanks :)

File: client/views/room/contextualBar/UserInfo/index.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ export const UserInfo = React.memo(function UserInfo({
 
 		<Margins block='x4'>
 			<UserCard.Username name={(showRealNames && name) || username || name} status={status} />
-			<Info>{customStatus}</Info>
+			<Info withTruncatedText={false}><MarkdownText content={customStatus} withRichContent={false}/></Info>
```
Comment (ggazzo) [inline client/views/room/contextualBar/UserInfo/index.js:67]:
```suggestion
			<Info withTruncatedText={false}><MarkdownText content={customStatus}  variant='inline' /></Info>
```

**Review by ggazzo — CHANGES_REQUESTED** [review]

File: client/views/room/contextualBar/UserInfo/index.js (hunk 1 above)
Comment (yash-rajpal) [reply client/views/room/contextualBar/UserInfo/index.js]: Done sir. :)

**Review by dougfabris — APPROVED** [review]

**Comment by yash-rajpal** [comment]
Done!

**Pull request closed without merging**